File: exemplos/js/01_versao.mjs
// Versão da plataforma sem chave nenhuma (GET /api/versao, escopo publico)
import { chamar, exigir } from "./_apoio.mjs";

const { status, objeto } = await chamar("/api/versao", { chave: "" });
exigir(status === 200, `esperava 200 em /api/versao, veio ${status}`);
exigir("versao" in objeto, `resposta sem o campo versao: ${JSON.stringify(objeto)}`);
console.log(`plat versao ${objeto.versao}`);

File: exemplos/python/01_versao.py
# Versão da plataforma sem chave nenhuma (GET /api/versao, escopo publico)
from _apoio import chamar, exigir

status, _cab, corpo = chamar("/api/versao", chave="")
exigir(status == 200, f"esperava 200 em /api/versao, veio {status}")
exigir("versao" in corpo, f"resposta sem o campo versao: {corpo}")
print(f"plat versao {corpo['versao']}")
